r/tests/testthat/test-dataset-write.R

```diff
@@ -139,6 +139,23 @@ test_that("Writing a dataset: Parquet->Parquet (default)", {
   )
 })
 
+test_that("Writing a dataset: `basename_template` default behavier", {
```

Comment:
We should have a test of providing `basename_template`, looks like we don't have one. Would you be interested in adding one here?

Reply:
Yes, I would.
I will try do so.

Reply:
I added testcases for `basename_template`.